wled00/bus_manager.h

```diff
@@ -7,6 +7,11 @@
 
 #include "const.h"
 
+//colors.cpp
+uint32_t colorBalanceFromKelvin(uint16_t kelvin, uint32_t rgb);
```

Comment:
Why did you move all three into .h? There is no need to do that, only move a needed one.

Reply:
Moved back